data-model/SDP-sample-records/codicological-unit_sample1.json

```diff
@@ -0,0 +1,143 @@
+{
+  "id": "ark:/21198/a310jfBd",
+  "type": "Codicological Unit",
+  "label": "Part 1",
+  "extent": ["f. 1-152"],
+  "dimensions": [{
+    "type": "average-folio",
+    "value": "320 x 250 x 92 mm"
+  }],
+  "support": {
+    "type": "Parchment",
+    "features": [],
+    "note": ["Quire 20 is paper."]
+  },
+  "condition": {
+    "note": [],
+    "features": []
+  },
+  "writing_and_hands": [{
+    "language": "Syriac",
+    "writing_system": "Syriac",
+    "script": "Estrangela",
+    "ink_color": "black",
+    "hand_note": ["Taw, Alaph, and Dalath/Resh all exhibit features of Serto."],
+    "textual_artifacts": ["ark:/21198/z1x07bdf"],
```

Comment:
This re-uses the field name ‘textual_artifacts’ with a different data format (it is used below to relate the unit to the artifacts but also includes sequence and layer information. Below it is an array of objects, here it is an array of strings). 

This might not matter since we will likely change the field name that relates units to artifacts to match our object relationship schema.

Reply:
We may not change the below field, so I will call this 'corresponding_textual_artifacts' for now.